src/main/java/org/isf/menu/gui/MainMenu.java

```diff
@@ -180,26 +176,25 @@ public MainMenu() {
 				Server.getInstance().login(myUser.getUserName(), myUser.getPasswd());
 				try {
 					Thread.sleep(500);
-				} catch (InterruptedException interruptedException) {
-					LOGGER.error(interruptedException.getMessage(), interruptedException);
+				} catch (InterruptedException e) {
+					e.printStackTrace();
 				}
 				new CommunicationFrame();
 				/*
-				 * Interaction communication= new Interaction();
-				 * communication.incomingChat(); communication.receiveFile();
+				 * Interaction communication= new Interaction(); communication.incomingChat(); communication.receiveFile();
 				 */
 			} catch (XMPPException e) {
 				String message = e.getMessage();
 				if (message.contains("SASL authentication DIGEST-MD5 failed")) {
-					if (ADMIN_STR.equals(myUser.getUserName())) {
-						LOGGER.error("Cannot use \"admin\" user, please consider creating another user under the admin group.");
+					if (myUser.getUserName().equals("admin")) {
+						logger.error("Cannot use \"admin\" user, please consider to create another user under the admin group");
 					} else {
-						LOGGER.error("Passwords do not match, please drop the XMPP user and login to OH again with the same user.");
+						logger.error("Passwords not matching, please drop XMPP user and login OH again with the same user");
 					}
 				} else if (message.contains("XMPPError connecting")) {
-					LOGGER.error("No XMPP Server seems to be running: set XMPPMODULEENABLED = false");
+					logger.error("No XMPP Server seems to be running: set XMPPMODULEENABLED = false");
 				} else {
-					LOGGER.error("An error occurs: {}", e.getMessage());
+					logger.error("An error occurs: {}", e.getMessage());
```

Comment:
correct form was one on develop

Reply:
yes; the messages are worded better in develop

Reply:
restored current informatici version